src/bin/deku_node.ml

```diff
@@ -113,32 +113,74 @@ let handle_ticket_balance =
       let state = Server.get_state () in
       let amount = Flows.request_ticket_balance state ~ticket ~address in
       Ok { amount })
-let node folder =
+
+let addr_to_string addr =
+  let open Lwt_unix in
+  match addr with
+  | ADDR_UNIX a -> a
+  | ADDR_INET (ia, p) -> Printf.sprintf "%s:%d" (Unix.string_of_inet_addr ia) p
+
+(* Initializes a socket and binds it to localhost:<port>
+   before returning it *)
+let create_socket port =
+  let open Lwt_unix in
+  let addr = ADDR_INET (Unix.inet_addr_loopback, port) in
+  let f_descr = socket PF_INET SOCK_DGRAM 0 in
+  let _ = Printf.sprintf "Bound to address: %s\n" (addr_to_string addr) in
+  f_descr
+
+let read_bytes buffer_size = failwith "undefined"
+
+(* handle_message should match on header? to parse the correct message which can be:
+          handle_block_level;
+          handle_received_block_and_signature;
+          handle_received_signature;
+          handle_block_by_hash;
+          handle_protocol_snapshot;
+          handle_request_nonce;
+          handle_register_uri;
+          handle_receive_user_operation_gossip;
+          handle_receive_consensus_operation;
+          handle_withdraw_proof;
+          handle_ticket_balance;
+          handle_trusted_validators_membership; *)
+let handle_message buffer_size = failwith "undefined"
+
+(* udp_node should definitively be in a dedicated module *)
+let udp_node folder =
+  (* those first lines are copied/pasted from previous `node` function. I think they are in charge of getting the current node state from the files in folder data/ *)
   let node = Node_state.get_initial_state ~folder |> Lwt_main.run in
```

Comment:
Quite not sure about these lines. They are important because I think it is the current state of the Node. But how can I ensure this is working? Any help would be welcome.

Reply:
Are you wondering _if_ these lines work or _how_ these lines work? In either case, I think looking at the implementation and compiling some questions for Eduardo or someone else knowledgeable on this part of the codebase would be productive.